### ISSUE TITLE
Odd Organ Behaviour After Surgery

### ISSUE THREAD

**Issue by BlackStar8107** [goonstation/goonstation#18427]
### Labels

[BUG]

### Description

After a surgery all of my organs reappeared on the little surgery tray and I keeled over almost immediatly afterwards.

### Steps to reproduce

0. Be on Cogmap 1
1. Place cyber-organs on the little movable tray ( Can move around and can be clicked to toggle breaks )
2. Have a borg perform surgery on human
3. Borg click + drags organs to put them into human from the tray
4. After surgery, have the human push the tray (bump into it)
5. Organs may have just reappeared on the table, human should keel over and die now ( Not instantly just crazy fast OXY )
( If you scan the body the organs are all still inside the human )
( I think it was only the internal organs that did this iirc the limbs did not reappear )

### Expected Behavior

Expected to not have the organs reappear and expected to not die 

### Additional Information & Screenshots

Should have happened during [this round](https://goonhub.com/rounds/175348).
Sadly I didn't take any screenshots.
Strangely I was actually completely fine, walking around and talking for a bit, I only keeled over after pushing the tray.
The organs that reappeared on the tray after I pushed it seemed to be real, I could move them around, pick them up and place them.
Initially we thought it was that I wasn't sutured but I definitely was sutured because I checked just after I died.

Basically no idea what could have even caused this. This is the second time I've had a weird tray issue like this.

**Comment by MylieDaniels** [comment]
Almost certainly related to the surgical_tray's vis_contents and contents being out of sync. I've seen cyborg click-dragging fail to sync the two in a few other bugs. It is likely that the surgical tray still had the organs in contents, and so did your organHolder. When the tray moved, however, it made sure that it's vis_contents and contents were aligned, so they visually reappeared and were removed from the duplicate location.
